types/recharts/index.d.ts

```diff
@@ -652,6 +652,7 @@ export interface ReferenceDotProps extends EventAttributes, Partial<Presentation
     y?: number | string;
     xAxisId?: string | number;
     yAxisId?: string | number;
+    label?: string | number;
```

Comment:
According to the reference label could be of type `String | Number | ReactElement | Function`. Please adjust the type of the property.